### PR TITLE
Upgrade dependencies across the board (#22)

### PR DESCRIPTION
**Pull request by sirthias**
This upgrade works for me locally with the latest versions of sbt-site and paradox.

Refs #22.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -113,7 +113,7 @@ lazy val theme = project("paradox-material-theme", file("theme"))
     libraryDependencies += "org.webjars" % "prettify" % "4-Mar-2013-1" % Provided,
     libraryDependencies +=
       Seq("animation", "base", "ripple", "rtl", "theme", "typography")
-        .foldLeft("org.webjars.npm" % "material__tabs" % "0.3.1" % Provided) {
+        .foldLeft("org.webjars.npm" % "material__tabs" % "0.37.1" % Provided) {
```
Comment (sirthias) [inline build.sbt]: I have no idea whether this does any good (or harm).
Simply realized that a versions have gone by since 0.3.1 here...

**Comment by joan38** [comment]
Could we move forward with this @jonas? Sounds like we are having dependency conflicts with Circe with other plugins

**Comment by kpritam** [comment]
Is this going to happen? I would be happy to contribute!

**Comment by mushtaq** [comment]
@kpritam 👍 

@jonas is it possible to share the commit rights with Pritam? We will be happy to contribute in ensuring the dependencies are up-to-date.

**Comment by mkurz** [comment]
Already up to date in main branch.

**Pull request closed without merging**